lib/kubeclient/http_error.rb

```diff
@@ -18,6 +19,7 @@ def to_s
   end
 end
 
+# Note: this IS NOT(!) a supported interface, and might change without a version bump
```

Comment:
why?  what are people supposed to do?  they need to know what exceptions to expect from us.